lib/commands/log.js

```diff
@@ -204,7 +205,10 @@ export default {
         log.debug(closeMsg);
       });
     });
-    await /** @type {AppiumServer} */ (this.server).addWebSocketHandler(pathname, wss);
+    await /** @type {AppiumServer} */ (this.server).addWebSocketHandler(
+      pathname,
+      /** @type {WSServer} */ (wss)
```

Comment:
should we use a typedef to import WSServer since it's not actually used as a var?